src/color/setting.js

```diff
@@ -462,7 +462,7 @@ p5.prototype.colorMode = function(mode, max1, max2, max3, maxA) {
  * 60x60 light green rect with black outline in center of canvas.
  * 60x60 soft green rect with black outline in center of canvas.
  * 60x60 red rect with black outline in center of canvas.
- * 60x60 dark fushcia rect with black outline in center of canvas.
+ * 60x60 dark fushsia rect with black outline in center of canvas.
```

Comment:
I think the correct spelling might be [fuchshia](https://en.wikipedia.org/wiki/Fuchsia_(color)). That's a tough-ie!

Reply:
Oh yeah.
Minor error from my side. Will fix it.
Thank you @stalgiag

Reply:
@stalgiag I think it's fuchsia and not fuchshia.

Reply:
Haha yes. Typos on typos.